python-frontend/src/main/java/org/sonar/python/semantic/v2/TypeShed.java

```diff
@@ -91,7 +91,7 @@ public class TypeShed {
   public TypeShed(ProjectLevelSymbolTable projectLevelSymbolTable) {
     // workaround to initialize supported python versions used in ClassSymbolImpl
     // TODO: remove once v2 types model will be populated from TypeShed bypassing conversion to symbols
-    org.sonar.python.types.TypeShed.builtinSymbols();
+    builtins = org.sonar.python.types.TypeShed.builtinSymbols();
```

Comment:
This is needed to ensure we convert builtins to type from the same set of `Symbol` objects.